packages/compiler/src/lib/decorators.ts

```diff
@@ -774,6 +774,9 @@ export function $withoutDefaultValues(context: DecoratorContext, target: Model)
 
 const listPropertiesKey = createStateSymbol("listProperties");
 
+/**
+ * @deprecated Use the `listsResource` decorator in `@typespec/rest` instead.
```

Comment:
Would add those entries to https://github.com/microsoft/typespec/issues/1873

Reply:
Will do

Reply:
Done